### ISSUE TITLE
Windows 7 - error on start

### ISSUE THREAD

**Issue by brcolow** [donnemartin/gitsome#10]
I installed gitsome with:

`pip3 install gitsome`

and it completed successfully.

When I then when to start `gitsome` in my ConEmu terminal, I got this result:

`C:\Python27\python.exe: No module named xonsh`

Not sure why it is trying to use Python27, though, as I installed it using Python 3.

Thanks!

**Comment by donnemartin** [comment]
Hi, could you post the output of:

     $ python --version

and:

     $ pip freeze

Are you installing in a `virtualenv`?

Do any of the `gh` commands work?

     $ gh trending

**Comment by zmurf** [comment]
I got the same problem.. trying to run it from powershell.

C:\tools\python\lib\site-packages> python --version
Python 3.4.3

C:\tools\python\lib\site-packages> python -m pip freeze
click==6.6
colorama==0.3.7
docopt==0.6.2
feedparser==5.2.1
gitsome==0.4.0
numpydoc==0.5
ply==3.4
prompt-toolkit==0.51
Pygments==2.1.3
pytz==2016.4
requests==2.8.1
six==1.10.0
uritemplate.py==0.3.0
wcwidth==0.1.6

gh commands doesnt work

**Comment by donnemartin** [comment]
>trying to run it from powershell.

@zmurf it looks like the [requirements](https://github.com/donnemartin/gitsome/blob/master/setup.py#L101-L113) are installed, do you get the same problem from `cmd` instead of powershell?

**Comment by celavi** [comment]
I am using Windows 7.  It's working like a charm. I am using cmder. I've also installed Pillow as stated in documentation.

- Install Python 3. I am using Choocolatey as installer (package manager).
- pip install Pillow
- pip install gitsome
- Add folder to %PATH% environment where gitsome resides. Mine is: C:\ProgramData\chocolatey\lib\python3\tools\Scripts;
- Open cmder and run gitsome
- Enyoy!

python -m pip freeze
click==6.6
colorama==0.3.7
docopt==0.6.2
feedparser==5.2.1
gitsome==0.4.0
numpydoc==0.5
Pillow==3.2.0
ply==3.4
prompt-toolkit==0.51
Pygments==2.1.3
pytz==2016.4
requests==2.8.1
six==1.10.0
uritemplate.py==0.3.0
wcwidth==0.1.6

![image](https://cloud.githubusercontent.com/assets/99165/15140832/35753f70-169d-11e6-965b-17197c475aac.png)

**Comment by zmurf** [comment]
This is embarrassing... I needed to restart my powershell session. Works perfectly now.

**Comment by donnemartin** [comment]
@celavi great, thanks for posting how you got set up!

@zmurf no prob, it happens to everyone :)

**Comment by brcolow** [comment]
Fixed by putting Python 3 ahead of Python 2 in $PATH. Sorry for the noise.

**Comment by donnemartin** [comment]
@brcolow we ran into a similar issue on a different project, please check out https://github.com/awslabs/aws-shell/issues/112 and let me know if that fixes the issue.

TLDR; Python doesn't like being installed in a path with spaces, which is possibly where your Python 3 is installed.

**Comment by donnemartin** [comment]
@brcolow did this [comment](https://github.com/donnemartin/gitsome/issues/10#issuecomment-218377361) help solve your problem?

**Comment by brcolow** [comment]
Sorry for not getting back to you sooner - yes. That fixes it. Thanks very
much.

On Tue, May 17, 2016 at 4:25 AM, Donne Martin <notifications@github.com>
wrote:

> @brcolow <https://github.com/brcolow> did this comment
> <https://github.com/donnemartin/gitsome/issues/10#issuecomment-218377361>
> help solve your problem?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/donnemartin/gitsome/issues/10#issuecomment-219690512>
>



-- 
Michael Ennen

**Comment by donnemartin** [comment]
Thanks for the update, glad to hear it's working for you.  I'll go ahead and close this issue.
